pyglmnet/pyglmnet.py

```diff
@@ -401,19 +401,19 @@ class GLM(BaseEstimator):
 
     Parameters
     ----------
-    distr : str
+    distr: str
```

Comment:
usually you have a space in the numpy docstring convention but I can live with this if it makes it consistent across the project.

Reply:
the docstring to API feature of sphinx doesn't understand this convention, so lets stick with no space

Reply:
hmm ... that's weird because in MNE we follow it.

Reply:
resolving for now